k8s/atrium-configmaps.yaml

```diff
@@ -11,3 +11,10 @@ metadata:
   name: parity-configmap
 data:
   parity_host: parity-service
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-configmap
+data:
+  client_url: https://proco-chain-demo4.unicef.io/
```

Comment:
This should not be in this one should be in a file like `atrium-configmaps-staging.yaml` As we don't want this locally or on Prod